adafruit_ble_berrymed_pulse_oximeter/adafruit_ble_transparent_uart.py

```diff
@@ -44,7 +50,7 @@ class TransparentUARTService(Service):
         buffer_size=64,
     )
 
-    def __init__(self, service=None):
+    def __init__(self, service: Optional[Service] = None):
```

Comment:
If you're following along with @FoamyGuy, this is actually typed as itself:

```suggestion
    def __init__(self, service: Optional["TransparentUARTService"] = None) -> None:
```

My understanding is that the way it works is that it will either initialize with what is given (itself) or it lazily creates itself (if provided `None`)

Reply:
I've never seen this syntax for specifying a type as a string. Going to trust you that it works! tough to google for...